src/core/src/Context.cpp

```diff
@@ -637,6 +641,15 @@ const pxr::TfToken& Context::getCesiumMdlPathToken() const {
     return _cesiumMdlPathToken;
 }
 
+bool Context::isCompressedTexturesSupported() const {
+    // Compressed textures do not work in Kit 105.0 or 105.0.1
+    if (_kitVersion.rfind("105.0", 0) == 0) {
```

Comment:
Is this even necessary? We just released version 0.11.0 which claims that versions less than 105.1 are unsupported.

Reply:
Oh good point... I'll remove it

Reply:
I think I worked on this PR before realizing that 0.11.0 wouldn't be backwards compatible with Kit 105.0

Reply:
In that case do we even need any of the code to pass the kit version up to the native layer?

Reply:
Nope, I just removed it all